src/hotspot/share/prims/jvmtiExport.cpp

```diff
@@ -383,6 +383,10 @@ JvmtiExport::get_jvmti_interface(JavaVM *jvm, void **penv, jint version) {
   if (Continuations::enabled()) {
     // Virtual threads support. There is a performance impact when VTMS transitions are enabled.
     java_lang_VirtualThread::set_notify_jvmti_events(true);
+    if (JvmtiEnv::get_phase() == JVMTI_PHASE_LIVE) {
+      ThreadInVMfromNative __tiv(JavaThread::current());
+      java_lang_VirtualThread::init_static_notify_jvmti_events();
+    }
   }
```

Comment:
Yes, this looks good now. Removing the `if (!java_lang_VirtualThread ::notify_jvmti_events())` check means the `init_static_notify_jvmti_events()` can still be called on subsequent calls to this method. So if `init_static_notify_jvmti_events()` was not called the first time (due to not being in the LIVE phase), then it can still be called on subsequent calls to this method if not in the LIVE phase.

Reply:
Okay. Thank you for the review, Chris,